routingv7/client.go

```diff
@@ -43,6 +43,7 @@ type service struct {
 }
 
 // An ErrorResponse reports the error caused by an API request.
+// nolint: errname // TODO: Consider breaking API change to follow XxxError convention.
```

Comment:
```suggestion
// nolint: errname // TODO(oscar.soderlund@einride.tech): Consider breaking API change to follow XxxError convention.
```

Reply:
Suggesting adding email to TODO:s for traceability

Reply:
IMO `git blame` should be enough

Reply:
this is a public repo, so would rather not put email in

Reply:
I'm reverting this suggestion. Had a further look, and Google was now discouraging this style. Possible that they put an issue tracker link in the TODO parenthesis instead.